.travis.yml

```diff
@@ -4,16 +4,29 @@ matrix:
   fast_finish: true
   include:
     - python: 3.6
+      dist: xenial
       env:
         - TOXENV=py36
         - AUTOGRAPH_INTEGRATION=1
+        - JAVA_HOME=/usr/lib/jvm/java-8-oracle
+      sudo: true
     - python: 3.7
       dist: xenial
       env:
         - TOXENV=py37
         - AUTOGRAPH_INTEGRATION=1
+        - JAVA_HOME=/usr/lib/jvm/java-8-oracle
       sudo: true
 
+before_install:
+    # Install oracle java 8, because jarsigner in openjdk 11.0.1+13 breaks
+    # integration tests.
+    # https://github.com/mozilla-releng/signingscript/issues/80
+    - sudo add-apt-repository -y ppa:webupd8team/java
```

Comment:
How about using Travis' `jdk_switcher`? pushapkscript uses it under xenial and it's compatible with oraclejdk8:

* https://github.com/mozilla-releng/pushapkscript/blob/91aaa35f170f6f263b2ebb1dca6c6f44ef6ea8bc/.travis.yml#L21
* https://docs.travis-ci.com/user/languages/java#switching-jdks-java-8-and-below-within-one-job

Reply:
nice, I wish I knew about this yesterday :)